config/holes.toml

```diff
@@ -2258,6 +2258,7 @@ category = 'Mathematics'
 experiment = 857
 links = [
     { name = 'Wikipedia', url = '//en.wikipedia.org/wiki/Rijndael_S-box' },
+    { name = "OEIS", url = "https://oeis.org/A354500"},
```

Comment:
Use style of other links:
- Add OEIS number
- Remove `https:`
- Use single quotes
- Add space before `}`
```diff
- { name = "OEIS", url = "https://oeis.org/A354500"},
+ { name = 'OEIS A354500', url = '//oeis.org/A354500' },
```
A354501 is also related, but I don't know if it's worth linking